docs/interactions/Receiving_and_Responding.md

```diff
@@ -12,22 +12,22 @@ For [Message Components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/) it includes ide
+| ---------------- | --------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
+| id               | snowflake                                                                                           | ID of the interaction                                                                                       |
+| application_id   | snowflake                                                                                           | ID of the application this interaction is for                                                               |
+| type             | [interaction type](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-interaction-type) | Type of interaction                                                                                         |
+| data?\*          | [interaction data](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-object-interaction-data) | Interaction data payload                                                                                    |
+| guild_id?        | snowflake                                                                                           | Guild that the interaction was sent from                                                                    |
+| channel_id?      | snowflake                                                                                           | Channel that the interaction was sent from                                                                  |
+| member?\*\*      | [guild member](#DOCS_RESOURCES_GUILD/guild-member-object) object                                    | Guild member data for the invoking user, including permissions                                              |
+| user?            | [user](#DOCS_RESOURCES_USER/user-object) object                                                     | User object for the invoking user, if invoked in a DM                                                       |
+| token            | string                                                                                              | Continuation token for responding to the interaction                                                        |
+| version          | integer                                                                                             | Read-only property, always `1`                                                                              |
+| message?         | [message](#DOCS_RESOURCES_CHANNEL/message-object) object                                            | For components and modal submissions opened in reply to components, message the components were attached to |
```

Comment:
The wording sounds like it could be improved

Reply:
I don't want the description to wrap. I can improve the wording if wrapping isn't a problem here